Procfile

```diff
@@ -1,4 +1,5 @@
 web: gunicorn opencraft.wsgi --timeout 60 --workers 4 --log-file -
 websocket: python3 websocket.py
 worker: python3 manage.py run_huey --no-periodic
+worker_shell: HUEY_QUEUE_NAME=opencraft_django_shell python3 manage.py run_huey --no-periodic
```

Comment:
How about just calling this "worker_low_priority"? Because you've said that you want to add the ability to trigger this from the GUI.